node/src/Commands.ts

```diff
@@ -3628,3 +3603,57 @@ export function createBZPopMin(
 ): command_request.Command {
     return createCommand(RequestType.BZPopMin, [...keys, timeout.toString()]);
 }
+
+/**
+ * Time unit representation which is used in optional arguments for {@link BaseClient.getex|getex} and {@link BaseClient.set|set} command.
+ */
+export enum TimeUnit {
+    /**
+     * Set the specified expire time, in seconds. Equivalent to
+     * `EX` in the VALKEY API.
+     */
+    seconds = "EX",
```

Comment:
Probably, all enum variant names should start with a capital letter
```suggestion
    Seconds = "EX",
```

Reply:
Update example too if you change it